File: mario_py/less/mario.py
from cs50 import get_int

# check for input between 1 to 8 inclusive
while True:
    n = get_int("Height: ")
    if n in range(1, 9):
        break

for i in range(1, n + 1):
    if i != n:
        print(" " * (n - i - 1), "#" * i)
    else:
        print("#" * i)

File: mario-pyramid_c/mario.c
#include <cs50.h>
#include <stdio.h>

int main(void)
{
    int input = 0;
    do
    {
        input = get_int("Input a number between 1 to 8, inclusive: ");
    }
    while (input < 1 || input > 8);
    
    
    for (int i = 1; i <= input; i++)
    {
        for (int j = 1; j <= input - i; j++)
        {
            printf(" ");
        }
        for (int k = 1; k <= i; k++)
        {
            printf("#");
        }
        
        for (int m = 1; m <= i; m++)
        {
            printf("#");
        }
        printf("\n");
    }
}